lib/wfsc/falco_wfsc_loop.m

```diff
@@ -8,15 +8,22 @@
 
 function [mp, out] = falco_wfsc_loop(mp, out)
 
-flagBreak = false;
 fprintf('\nBeginning Trial %d of Series %d.\n', mp.TrialNum, mp.SeriesNum);
 mp.thput_vec = zeros(mp.Nitr+1, 1);
 
+flagBreak = false;
+
 for Itr = 1:mp.Nitr
     
     
     %% Bookkeeping
-    fprintf(['WFSC Iteration: ' num2str(Itr) '/' num2str(mp.Nitr) '\n' ]);
+    fprintf(['WFSC Iteration: ' num2str(Itr) '/' num2str(mp.Nitr) ', ' datestr(now) '\n' ]);
+    
+    % update subdir for scicam images
+    if isfield(mp, 'tb') && isfield(mp, 'testbed') && isequal(upper(mp.testbed), 'OMC')
+        mp.tb.sciCam.subdir = ['Series_' num2str(mp.SeriesNum) '_Trial_' num2str(mp.TrialNum) '_It_' num2str(Itr)];

```

Comment:
@davidsmarx I think this is the last thing to change so that the the mp.tb stuff goes into another function (via a handle) or otherwise.